src/app/core/services/github.service.ts

```diff
@@ -317,11 +317,14 @@ export class GithubService {
    * @return Observable<SessionData> representing session information.
    */
   fetchSettingsFile(): Observable<SessionData> {
-    return from(octokit.repos.getContents({owner: MOD_ORG, repo: DATA_REPO, path: 'settings.json',
-      headers: GithubService.IF_NONE_MATCH_EMPTY})).pipe(
-        map(rawData => JSON.parse(atob(rawData['data']['content']))),
-      catchError(err => throwError('Failed to fetch settings file.'))
-    );
+    const github_api_link = `https://api.github.com/repos/${MOD_ORG}/${DATA_REPO}/contents/settings.json`;
```

Comment:
Do we still need to remove the usage of the API, since we're only fetching the settings file after the OAuth process has completed ?

Reply:
SFLR, yes we wouldn't need to remove the usage of the API call, let me fix that